oauth2_http/java/com/google/auth/oauth2/DefaultPKCEProvider.java

```diff
@@ -36,40 +36,16 @@
 import java.security.SecureRandom;
 import java.util.Base64;
 
+/**
+ * Implements PKCE using only the Java standard library. See https://www.rfc-editor.org/rfc/rfc7636.
+ *
+ * <p>go/client-auth-java-oauth-pkce
```

Comment:
we usually don't include this to javadoc, i meant the link to Google docs on PKCE that you have referenced in your doc

Reply:
Ah gotcha!